code/game/objects/items/holosign_creator.dm

```diff
@@ -155,6 +155,16 @@
 	. = ..()
 	register_context()
 
+/obj/item/holosign_creator/atmos/interact_with_atom(atom/interacting_with, mob/living/user, list/modifiers)
+	. = ..()
+	if(. != ITEM_INTERACT_SUCCESS)
+		return
+	var/turf/target_turf = get_turf(interacting_with)
+	var/obj/machinery/door/firedoor/firelock = locate() in target_turf
+	if(!firelock)
+		return
+	firelock.open()
```

Comment:
```suggestion
	firelock?.open()
```

Reply:
considered it, both options are fine but sure.